### DISCUSSION TITLE
Zksync Debit Card

### DISCUSSION THREAD

**Discussion by kamranbash** [zksync/credo#77] [Ideas]
It would be nice to have a Zksync Debit card that lets users pay using their assets on Zksync Era. A direct partnership with Visa would be phenomenal.

**Comment by impnahid23213** [comment]
I would assume a safe.global wallet would play a role

**Reply by enkazweb** [reply]
safe is important

**Comment by sifahomma** [comment]
There is already debit cards that accept crypto, but I don't think this will be helpful to zksync

**Reply by kamranbash** [reply]
It will increase network utility Ser

**Reply by enkazweb** [reply]
yes its required

**Comment by mrkay100** [comment]
Could be a good Idea but who will be regulating this. Crypto is meant to be decentralised

**Reply by kamranbash** [reply]
This will require kyc obviously, decentralised does not mean not regulated though.

**Comment by critdmg** [comment]
not need i think

**Reply by enkazweb** [reply]
why

**Comment by Armankhan566** [comment]
It's good but AirDroid then

**Comment by enkazweb** [comment]
need crypto credit card..
